mongo/integration/csot_prose_test.go

```diff
@@ -20,7 +23,61 @@ func TestCSOTProse(t *testing.T) {
 	mt := mtest.New(t, mtest.NewOptions().CreateClient(false))
 	defer mt.Close()
 
-	mt.Run("server selection", func(mt *mtest.T) {
+	mt.RunOpts("1. multi-batch writes", mtest.NewOptions().MinServerVersion("4.4"), func(mt *mtest.T) {
+		// Test that multi-batch writes do not refresh the Timeout between batches.
+		if testing.Short() {
+			t.Skip("skipping integration test in short mode")
+		}
```

Comment:
```suggestion
```

IIUC this is no longer necessary. GODRIVER-2184 applies `testing.Short()` to all integration tests.

Reply:
Good catch. I totally forgot about that improvement.